contribution.md

```diff
@@ -0,0 +1,59 @@
+# Project;
+ Current name: KUWINDA(Trademark Pending)
+ Kind: FullStack Application(built using next.js)
+ Creator: E-man
+ Contributors: Ladu-man, Kon-AK,
+ Executive Supervisor: COCO.
+ Main gitHub branch: https://github.com/Emmanuel-Melon/kuwinda.git 
+ Database: SQL database provided by planetscale
+ 
+## Frameworks used in project;
+ next.js(Entire App)
```

Comment:
Markdown also allows you to create lists, this section should look a bit lile this:
- Next.js 
- React.js